src/components/QuranReader/types.ts

```diff
@@ -28,4 +28,27 @@ export enum QuranFont {
   QPCHafs = 'qpc_uthmani_hafs',
 }
 
+export enum MushafLine {
+  FifteenLines = '15_lines',
+  SixteenLines = '16_lines',
+}
```

Comment:
IMO this should be MushafLines, each property of the enum is a group of lines, not a single line